src/server/local/startMongo.bat

```diff
@@ -1 +1 @@
-"C:\Program Files\MongoDB\Server\3.4\bin\mongo.exe"
\ No newline at end of file
+"C:\Program Files\MongoDB\Server\3.4\bin\mongo.exe"
```

Comment:
Probably we should put all these Windows batch files in a subdirectory of `local`, with a directory name such as `bat` or `windows_bat`.